src/sentry/auth/password_validation.py

```diff
@@ -135,7 +135,7 @@ class NumericPasswordValidator:
     Validate whether the password is alphanumeric.
     """
 
-    def validate(self, password):
+    def validate(self, password, user=None):
         if password.isdigit():
             raise ValidationError(
                 _("This password is entirely numeric."), code="password_entirely_numeric"
```

Comment:
This can be more clear, imo.
```suggestion
                _("Password cannot be entirely numeric."), code="password_entirely_numeric"
```

Reply:
This string comes from the actual validator:
https://github.com/getsentry/sentry/blob/19781a29169c5603a7cf51a15f41d8a461f2809d/src/sentry/auth/password_validation.py#L141
We vendored it from the actual Django:
https://github.com/django/django/blob/e7e8eb44a30bcab004a582760752b5eb3fcd6e91/django/contrib/auth/password_validation.py#L261
So I would keep it as is.